Testing/Fakes/QueueFake.php

```diff
@@ -119,7 +126,7 @@ public function connection($value = null)
      */
     public function size($queue = null)
     {
-        return 0;
+        return $this->size;
```

Comment:
this doesn't respect the `queue` argument. Might want to do something like this:
```
$jobs = collect($this->jobs)->flatten(1);

if (!$queue) {
    return $jobs->count();
}

return $jobs->filter(function($value, $key) use ($queue) {
    return $value['queue'] === $queue;
})->count();
```